src/interface/src/app/plan/create-scenarios/create-scenarios.component.html

```diff
@@ -2,6 +2,7 @@
   <div class="create-scenarios-panel-content" [@expandCollapsePanelContent]="panelExpanded ? 'opaque' : 'transparent'">
     <app-create-scenarios-intro *ngIf="planningStep === 1"></app-create-scenarios-intro>
     <app-set-priorities *ngIf="planningStep === 2" (changeConditionEvent)="changeConditionEvent.emit($event)"></app-set-priorities>
+    <app-constraints-panel *ngIf="planningStep === PlanStep.SetConstraints" (changeConstraintsEvent)="changeConstraintsEvent.emit($event)"></app-constraints-panel>
```

Comment:
do you mind changing lines 3 and 4 to use the PlanStep type as well?

Reply:
done!